### DISCUSSION TITLE
Abogabot

### DISCUSSION THREAD

**Discussion by NestorMyrie** [Launch-X-Latam/MisionFrontEnd#93] [Q&A]
### Hola buenas tardes explorers, tengo una duda,
en la práctica de introducción al Frontend  [AbogaBot](https://github.com/Launch-X-Latam/MisionFrontEnd/blob/main/01%20-%20INTRO/practicas/README.md), debemos  crear un WireFrame y luego el diseño completo con los requerimientos del bot?

**Comment by PacoMedGar** [comment]
Veremos el proceso hoy compañero, empezaremos con crear un wireframe y veremos paso a paso los procesos para crear un proyecto

**Reply by NestorMyrie** [reply]
Excelente gracias!!!
